src/modules/notifications/controllers/notifications.controller.ts

```diff
@@ -54,4 +79,120 @@ export class NotificationsController {
 		);
 		return { status: `updated ${amountUpdated} notifications`, total: amountUpdated };
 	}
+
+	@UseGuards(ApiKeyGuard)
+	@Post('check-new-notifications')
+	public async checkNewNotifications(
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		@Headers('apiKey') apiKey: string
+	): Promise<{
+		status: string;
+		notifications?: Partial<Record<NotificationType, number>>;
+		total: number;
+	}> {
+		const accessStartsNotifications = await this.sendAccessStartNotifications();
```

Comment:
🤔 shouldn't we filter these based on the logged in userId? We can't show notifications for a not-logged in user, since the frontend cannot check the recipient unless logged in. 
In that case the PROXY_API_KEY kan be discarded in favor of the loggedIn guard?
Maybe I'm missing something...

Reply:
This api call will be triggered by a cron job somewhere in openshift. We do this as a route for 2 reasons:
* We can test these easily by triggering them using postman
* They can be triggered once on a single pod if the proxy every has to be load balanced across multiple pods

Reply:
Okay. I should add a comment above this controller function that this one is cron-triggered

Reply:
done: https://github.com/viaacode/hetarchief-proxy/pull/72/files#diff-6e6f91f8cfa498dcec22d8053887b08861fb6df06a3532bd0725d12433e04cbaR82-R89